INSTALL.md

```diff
@@ -37,6 +37,10 @@ cd tools/toolchain/
 Sub-points here discuss prerequisites needed to build CP2K. Copies of the
 recommended versions of 3rd party software can be downloaded from <https://www.cp2k.org/static/downloads/>.
 
+Generally, CP2K supports only one version for each of its dependencies.
+These are defined by the [toolchain scripts](./tools/toolchain/scripts/).
+Other versions might work too, but we don't test them. So, your mileage may vary.
+
```

Comment:
I hoped we'd provide the information in a more user-friendly, especially when users want to reuse for example minor versions of libraries they have already installed. And for things like BLAS/LAPACK/ScaLAPACK this also does not generally apply, does it?

Reply:
I think, it's transparent and honest. We only test one combination of versions and we shouldn't make any further promises.

> And for things like BLAS/LAPACK/ScaLAPACK this also does not generally apply, does it?

Same here. We only test one combination of versions. I do agree that these libraries are very stable and there is a high chance that other versions will work too, but we can't be certain.